File: proxy/tests/script_engine/query_to_string.js
/// <reference path="../../script_libs/js/index.d.ts" />
/** @type {Extension} */
const query_to_string = {
  request(flow) {
    flow.request.body.text = flow.request.url.searchParams.toString();
  }
};
globalThis.extensions = [query_to_string];

File: proxy/tests/script_engine/query_to_string.lua
pcall(require, "../../script_libs/lua/roxy.lua")
---@type Extension
local query_to_string = {
	request = function(flow)
		flow.request.body.text = tostring(flow.request.url.search_params)
	end,
}
Extensions = { query_to_string }
